src/sparseml/pytorch/sparsification/__init__.py

```diff
@@ -23,3 +23,4 @@
 
 from .info import *
 from .pruning import *
+from .quantization import *
```

Comment:
should only import the modifier here

Reply:
I think that will make the imports very long for the user,  `from sparseml.pytorch.sparsifiction.quantization import QuantizationModifier` but made the change